index.js

```diff
@@ -69,6 +70,7 @@ module.exports = {
     'primer/responsive-widths': true,
     'primer/spacing': true,
     'primer/typography': true,
+    'primer/utilities': null,
```

Comment:
I turned this off by default as it's pretty specific circumstances you'd want to turn it on